.eslintrc.json

```diff
@@ -24,6 +24,7 @@
     "no-underscore-dangle": 0,
     "no-console": 0,
     "no-unused-vars": 0,
+    "no-constant-condition": 0,
```

Comment:
This was specifically because ```
  try {
    while (true) {
 ...
``` was stoping webpack from even starting. I think its setup to the point where we won't have to touch it too much more.